### DISCUSSION TITLE
如何知道Python物件可用的方法？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#110] [General]
若想知道Python中的物件可以使用什麼方法，只要先打此「資料型態=型別=data type」，接者在後面打一個點點(.)，再按下鍵盤上的tab，便能知道此物件能使用的「函數=函式=方法=function。」

**Comment by chun1203** [comment]
謝謝實用的分享！

**Comment by enoch-lightning** [comment]
不客氣！之後你也可以分享。:smile:
